airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/CdcPostgresSourceComprehensiveTest.java

```diff
@@ -356,6 +356,14 @@ protected void initTests() {
             .addExpectedValues("99999", "5.1", "0", null)
             .build());
 
+    addDataTypeTestData(
+        TestDataHolder.builder()
+            .fullSourceDataType("numeric(13,4)")
```

Comment:
this needs the following row:
.sourceType("decimal") 

See the other data types for example